sklearn/gaussian_process/kernels.py

```diff
@@ -79,6 +79,36 @@ class Hyperparameter(namedtuple('Hyperparameter',
         changed during hyperparameter tuning. If None is passed, the "fixed" is
         derived based on the given bounds.
 
+    Examples
+    --------
+    >>> from sklearn.gaussian_process.kernels import ConstantKernel, RBF
+    >>> from sklearn.datasets import make_friedman2
+    >>> from sklearn.gaussian_process import GaussianProcessRegressor
+    >>> from sklearn.gaussian_process.kernels import Hyperparameter
+    >>> X, y = make_friedman2(n_samples=50, noise=0, random_state=0)
+    >>> kernel = ConstantKernel(constant_value=1.0,
+    ...    constant_value_bounds=(0.0, 10.0)) * RBF(length_scale=0.5,
+    ...    length_scale_bounds=(0.0, 10.0)) + RBF(length_scale=2.0,
+    ...    length_scale_bounds=(0.0, 10.0))
+    >>> for hyperparameter in kernel.hyperparameters: print(hyperparameter)
+    Hyperparameter(name='k1__k1__constant_value', value_type='numeric',
+    bounds=array([[ 0., 10.]]), n_elements=1, fixed=False)
+    Hyperparameter(name='k1__k2__length_scale', value_type='numeric',
+    bounds=array([[ 0., 10.]]), n_elements=1, fixed=False)
+    Hyperparameter(name='k2__length_scale', value_type='numeric',
+    bounds=array([[ 0., 10.]]), n_elements=1, fixed=False)
+    >>> params = kernel.get_params()
+    >>> for key in sorted(params): print("%s : %s" % (key, params[key]))
```

Comment:
```suggestion
    >>> for key in sorted(params): print(f"{key} : {params[key]})
```

Reply:
You can even put it on 2 lines as in the example before.

Reply:
Utilizing f strings over C-style makes sense here.